### PR TITLE
Slurm accounting docs

### PR DESCRIPTION
**Pull request by sean-smith**
Signed-off-by: Sean Smith <seaam@amazon.com>

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/static/02-tutorials/slurm-accounting/pcluster/cluster-config.yaml [hunk 1]
```diff
@@ -1,39 +1,26 @@
...
 HeadNode:
-  InstanceType: t3.xlarge
+  InstanceType: c5.xlarge
   Networking:
-    SubnetId: subnet-03ffbdaeee8264db7
-    AdditionalSecurityGroups:
-      - sg-0fec342c8a6f2a5ea
-  Ssh:
-    KeyName: aws_rsa
+    SubnetId: subnet-846f1aff
```
Comment (charlesg3) [inline docs/static/02-tutorials/slurm-accounting/pcluster/cluster-config.yaml:4]: seems like we shouldn't have this magic number in the config

File: docs/content/02-tutorials/02-slurm-accounting.md [hunk 2]
```diff
@@ -64,7 +100,7 @@ Next we'll enable a known script that will install slurm accounting on the HeadN
 - Under the `On Configured` option, Choose the `Multi-Script Runner` which has some pre-programmed scripts in it
 - In the search box choose `Slurm Accounting`
 - Fill in the values for the `Secret ARN` and `RDS Endpoint` from the CloudFormation output
-- Under `IAM Policies` add `arn:aws:iam::aws:policy/SecretsManagerReadWrite` so that the HeadNode can access the password to the database generated by the CloudFormation script. Be sure to actually click `Add` so that it is added to the list.
+- Under `IAM Policies` add the arn from the CloudFormation Stack output `SecretsManagerPolicy` so that the HeadNode can access the password to the database. Be sure to actually click `Add` so that it is added to the list.
```
Comment (charlesg3) [inline docs/content/02-tutorials/02-slurm-accounting.md:103]: Was this changed to include the output?

File: docs/static/02-tutorials/slurm-accounting/pcluster/cluster-config.yaml (hunk 1 above)
Comment (sean-smith) [reply docs/static/02-tutorials/slurm-accounting/pcluster/cluster-config.yaml:4]: agreed

**Review by charlesg3 — COMMENTED** [review]
Should we bring back the example template at the bottom?

**Comment by sean-smith** [comment]
@charlesg3 I don't see why

**Review by charlesg3 — APPROVED** [review]

**Pull request merged**